SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -185,6 +187,41 @@ void USpatialNetDriver::OnMapLoaded(UWorld* LoadedWorld)
 	Connect();
 }
 
+void USpatialNetDriver::OnLevelAddedToWorld(ULevel* LoadedLevel, UWorld* OwningWorld)
+{
+	// If we have authority over the GSM when loading a sublevel, make sure we have authority
+	// over the actors in the sublevel.
+	if (GlobalStateManager != nullptr)
```

Comment:
Should we `check(GlobalStateManager)`? Is it valid to get here without it?

Reply:
Yes it is valid since you can be loading levels before you created the GlobalStateManager.